app/assets/stylesheets/regions/mastheads/slider-masthead.scss

```diff
@@ -1 +1,70 @@
-/* New slider masthead styles will go here */
\ No newline at end of file
+/*
+ * Slider v201611
+ */
+div.slider.version-201611 {
+  margin: 0 auto;
+  width: 1130px;
+  height: 440px;
+
+  div.grid {
+    .column {
+      float: left;
+      position: relative;
+      height: 440px;
+      overflow: hidden;
+      font-family: $font-stack-heading;
+
+      &.image {
+        width: 780px;
+
+        div.photo-by {
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          padding: 16px;
+          font-size: 16px;
+          color: $white;
+          background-color: rgba(56,56,56,0.90);
+        }
+      }
+
+      &.header {
+        width: 350px;
+        color: $white;
+
+        div.aligned {
+          position: absolute;
```

Comment:
@homeofmatt I agree that complexity/nesting should be avoided, but that comes from the HTML, so I don't think it can be avoided here. I'd prefer the Sass acknowledge the complexity that's beyond it control rather than hide it.